src/main/plots/ati_data_plots.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+import pandas as pd
+import matplotlib.pyplot as plt
+
+from src.main.util import consts
+from src.main.plots import consts as plot_consts
+from src.main.plots.plots_common import save_plot, add_fragments_length_plot, get_short_name, EVENT_DATA_COL, \
+    TIMESTAMP_COL, FRAGMENT_COL
+
+log = logging.getLogger(consts.LOGGER_NAME)
+
+
+def __create_ati_events_plot(ax: plt.axes, df: pd.DataFrame, event_data: list, event_colors: dict, title: str):
+    add_fragments_length_plot(ax, df)
+    for event_data in event_data:
+        event_data_df = df.loc[df[EVENT_DATA_COL] == event_data]
+        ax.scatter(event_data_df[TIMESTAMP_COL], event_data_df[plot_consts.FRAGMENT_LENGTH_COL],
```

Comment:
It is very similar to `add_fragments_length_plot` from the `plots_common` file. Maybe should we change `add_fragments_length_plot`: add default parametrs `color=plot_consts.FRAGMENT_LENGTH_COLOR`, `s=plot_consts.SMALL_SIZE` and empty label?